interface/main/tabs/js/user_data_view_model.js

```diff
@@ -133,3 +133,9 @@ function logout()
     top.restoreSession();
     document.getElementById("logoutinnerframe").src=webroot_url+"/interface/logout.php";
 }
+
+function timeoutLogout()
+{
+    top.restoreSession();
+    document.getElementById("logoutinnerframe").src=webroot_url+"/interface/logout.php?timeout=1";
+}
```

Comment:
Because this file changed, $v_js_includes in version.php should be updated.

Reply:
good point! will do.